app/blueprints/assessments/routes.py

```diff
@@ -949,6 +951,47 @@ def assignment_overview(fund_short_name: str, round_short_name: str):
 )
 @check_access_fund_short_name_round_sn
 def fund_dashboard(fund_short_name: str, round_short_name: str):
+    # Create a unique key for the session based on the fund short name and round short name
+    session_key = f"filter_params_{fund_short_name.upper()}_{round_short_name.upper()}"
+
+    # If there are no query arguments in the request
+    if not request.args:
+        # Check if there are saved filter parameters in the session
+        if session_key in session and session[session_key] is not None:
+            filter_params = session[session_key]
+            # If the 'clear_filters' flag is set, restore all saved filter parameters
+            if "clear_filters" in filter_params:
+                request.args = ImmutableMultiDict(session[session_key])
+            # Otherwise, merge saved filter parameters with existing query arguments
+            else:
+                request.args = ImmutableMultiDict(
+                    {
+                        **request.args,
```

Comment:
`request.args` must be None / empty by line 958 , so I think this is redundant?

Reply:
When a filter is applied the request.args are not null at this point, e.g. would contain the filters being applied, ImmutableMultiDict([('search_term', ''), ('status', 'NOT_STARTED'), ('assigned_to', 'All'), ('filter_by_tag', 'ALL')])